modules/monitoring-enabling-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * monitoring/user-workload-monitoring.adoc
+
+[id="enabling-user-workload-monitoring_{context}"]
+= Enabling user workload monitoring
+
+You can enable user workload monitoring using the cluster monitoring ConfigMap.
+
+.Prerequisites
+
+* Make sure you have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section.
+
+.Procedure
+
+. Start editing the `cluster-monitoring-config` ConfigMap:
++
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Set the `techPreviewUserWorkload` setting to `true` under `data/config.yaml`:
++
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    techPreviewUserWorkload:
+      enabled: true
+----
+
+. Save the file to apply the changes. The user workload monitoring is enabled automatically.
```

Comment:
It is better to add
you can check pods under openshift-user-workload-monitoring namespace
# oc -n openshift-user-workload-monitoring get pod
NAME                                   READY   STATUS    RESTARTS   AGE
prometheus-operator-85bbb7b64d-7jwjd   1/1     Running   0          3m24s
prometheus-user-workload-0             5/5     Running   1          3m13s
prometheus-user-workload-1             5/5     Running   1          3m13s

Reply:
agreed, this is a good verification :+1: